src/style.css

```diff
@@ -1,3 +1,81 @@
+:root {
+  --blue-color: #00acdc;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
+  font-family: Roboto, sans-serif;
+}
+
+header {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-sizing: border-box;
+}
+
+.nav_menu {
+  display: flex;
+  justify-content: flex-start;
+  text-transform: uppercase;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav_menu a {
+  margin: 0 0 0 20px;
```

Comment:
Be consistent with your vertical margins. It's better to add only top or only bottom margin, but not both. In this case, you can move `margin: 0 0 0 20px;` to `.nav_menu li`.